pkg/services/accesscontrol/accesscontrol.go

```diff
@@ -47,6 +49,12 @@ type Options struct {
 	ReloadCache bool
 }
 
+type SearchOptions struct {
+	ActionPrefix string
+	Action       string
```

Comment:
thought: could we just have `ActionPrefix` as an option, and if the user wants to filter by a specific action then they pass the full action in there? The chances of one action being a prefix to another one are pretty slim, no?

Reply:
As we discussed, I'm thinking that we'll remove the `actionPrefix` possibility altogether once the PoC doesn't need it, this is why I'd prefer keeping both.